test/test_torch.py

```diff
@@ -17713,6 +17715,7 @@ def inner(self, device, dtype):
     ('rsqrt', '', lambda t, d: _small_3d(t, d) + 1, lambda t, d: [], 1e-2, 1e-5, 1e-4, _float_types_no_half),
     ('sinh', '', lambda t, d: _small_3d(t, d).clamp(-1, 1), lambda t, d: [], 1e-3, 1e-5, 1e-5, _float_types),
     ('tan', '', lambda t, d: _small_3d(t, d).clamp(-1, 1), lambda t, d: [], 1e-3, 1e-5, 1e-5, _float_types),
+    ('tan', 'complex', lambda t, d: _small_3d(t, d), lambda t, d: [], 1e-3, 1e-5, 1e-5, _complex_types)
```

Comment:
I am surprised that `_complex_types` did not exist yet here. How do we test the other methods that already support complex?

Reply:
Unfortunately there are no existing tests for these ops for complex dtypes. we should add it soon though ...